### PR TITLE
Validate fields length

### PR DESCRIPTION
**Pull request by perchespierri**
Closes: https://github.com/comarev/comarev/issues/46

#### What?

- Creates model validations for cnpj, cpf nad cellphone
- Adds length tests for these fields

#### Why?

- These validations were necessary in order to register new users
- CPF fields must have 11 digits
- CNPJ fields must have 14 digits
- User cellphone phone fields must have 15 digits
- Company phone fields must have 14 digits

#### How to test it?

- Run bundle exec rspec after starting the server

### CODE REVIEW

**Comment by perchespierri** [comment]
> Hello, @perchespierri thanks for this PR! I'm just checking if we can add some examples to tell other contributors why these validations. Also, can you please link the issue that this pr is for? Just add a line at the very top of pr's description box, like: `Closes: <issue-link>`

@m-pereira I've made the suggested changes and comments.

File: .env.example [hunk 1]
```diff
@@ -1,10 +0,0 @@
-DEVISE_JWT_SECRET_KEY=
```
Comment (m-pereira) [inline .env.example:1]: This file is an example of the env vars we have for this project, it cannot be deleted.

**Review by m-pereira — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-version: '3.9'
+version: '3.3'
```
[the remark this answers is not in the document]
Comment (perchespierri) [reply docker-compose.yml]: It was necessary to function propperly in my computer. I'll change it back.

File: .env.example (hunk 1 above)
Comment (perchespierri) [reply .env.example:1]: Ok. Sorry for that.

**Review by m-pereira — APPROVED** [review]

**Comment by m-pereira** [comment]
Hey, @perchespierri thanks for your pr, I think that's it, can you please check the rubocop offenses?

**Comment by perchespierri** [comment]
> Hey, @perchespierri thanks for your pr, I think that's it, can you please check the rubocop offenses?

Done! Thank you

**Comment by m-pereira** [comment]
@allcontributors add @perchespierri for code

**Pull request merged**